howsmyssl.go

```diff
@@ -136,11 +136,22 @@ func main() {
 	}
 	oa := newOriginAllower(blockedOrigins, hostname, gclog, expvar.NewMap("origins"))
 
+	staticHandler := http.NotFoundHandler()
+	webHandler := func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Oh, and I guess the type of webHandler parameter should be `http.HandlerFunc`.

Alternatively, it could be http.Handler and you use conversions to `http.HandlerFunc` inside the if-statement, and `http.NotFoundHandler`, elsewhere.